src/medias/images/UGT_Asset_Name.tsx

```diff
@@ -1,19 +1,85 @@
 import React from "react";
 
-export interface AssetNameProps extends React.SVGProps<SVGSVGElement> {
-  width?: number;
-  height?: number;
-}
+export interface AssetNameProps extends React.SVGProps<SVGSVGElement> {}
 
-export const AssetName: React.FunctionComponent<AssetNameProps> = ({ width = 12, height = 80, ...props }) => {
+export const AssetName: React.FunctionComponent<AssetNameProps> = ({ ...props }) => {
   return (
-    <svg width="115" height="23" viewBox="0 0 115 23" fill="none" xmlns="http://www.w3.org/2000/svg">
+    <svg
+      width="97"
+      height="29"
+      viewBox="0 0 97 29"
+      fill="none"
+      xmlns="http://www.w3.org/2000/svg"
+      // {...props}
```

Comment:
```suggestion
      {...props}
```
Sorry, haven't noticed up until now 😅 If we accept the SVG props we also have to apps them along, otherwise this can lead to unexpected behavior (like with the Flags earlier where I forgot the same thing)